bpf/aya-bpf/src/programs/sk_buff.rs

```diff
@@ -65,6 +66,79 @@ impl SkBuffContext {
         }
     }
 
+    /// Writes bytes from the SKB context to the given bytes slice.
```

Comment:
```
Reads some bytes from the packet into the specified buffer, returning how many bytes were read.

Starts reading at `offset` and reads at most `dest.len()` bytes.
```

Reply:
I added a bit more to the second sentence, it might be either `dst.len()` or `self.len() - offset`.